app/presenters/menu/default_menu.rb

```diff
@@ -2,6 +2,16 @@
 module Menu
   class DefaultMenu
     class << self
+      def graphql_menu_section
+        return nil unless Rails.env.development?
+        Menu::Section.new(
+          :graphql,
+          N_("GraphQL"),
+          'fa fa-lightbulb-o',
+          [Menu::Item.new('graphql', N_('Explorer'), '', {:feature => 'utilization'}, '/graphql_explorer')]
```

Comment:
@AparnaKarve making that identifier `''` caused the menu item to disappear - do you have any recommendations?

Reply:
@imtayadeway It looks like, for the menu to render, the identifier needs to be in miq_product_features.yml.

Since this is for development mode only, I suppose we could take a few liberties.
How about we specify `product` as the feature identifier?
(I believe it is associated with accessing the `Product Website` page; also sounds generic enough to be used for this task)
```rb
Menu::Item.new('graphql', N_('Explorer'), 'product', {:feature => 'product'}, '/graphql_explorer'),
```

Or

We could actually create a new feature id called `development` for development specific tasks.

Reply:
Sounds good to me. Thanks @AparnaKarve !